test/expected/drop_owned.out

```diff
@@ -0,0 +1,82 @@
+\c single :ROLE_SUPERUSER
+CREATE SCHEMA hypertable_schema;
+GRANT ALL ON SCHEMA hypertable_schema TO :ROLE_DEFAULT_PERM_USER;
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+CREATE TABLE hypertable_schema.test1 (time timestamptz, temp float, location int);
+SELECT create_hypertable('hypertable_schema.test1', 'time', 'location', 2);
+NOTICE:  adding NOT NULL constraint to column "time"
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+INSERT INTO hypertable_schema.test1 VALUES ('2001-01-01 01:01:01', 23.3, 1);
```

Comment:
Can we call the tables `perm_user` and `superuser` or something to easily see that the right one is dropped?